### PR TITLE
Add new 180px icons

### LINKED ISSUE #7314: Add 180px icons, and list them in the manifests
**Is your feature request related to a problem? Please describe.**
iOS now expects a 180px icon.  We've got 180px icons, we just need to add them to Simorgh. (Jim has the PNG files)

**Describe the solution you'd like**
* Add the 180px PNG files to the relevant locations in the Simorgh repo
* Update the services' manifest files to point to them
* Update the metadata container to surface the path of the 180px icons
* Update tests

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves #7314

**Overall change:**
Adds 180px brand icons

**Code changes:**

* Adds icon PNGs
* Update manifest files
* Update metadata container to list new file size
* Update tests

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by jimjohnsonrollings** [comment]
Blocked awaiting more image files from UX

**Comment by jimjohnsonrollings** [comment]
Unblocked

File: public/afaanoromoo/manifest.json
Comment (HarveyPeachey) [inline public/afaanoromoo/manifest.json:35]: Are these images meant to be directly viewable? Because I'm getting the error message `The requested resource could not be found`

Comment (chris-hinds) [reply public/afaanoromoo/manifest.json:35]: They will be available on that url once this PR gets merged. Merging the PR will upload these new assets along with the other static assets.

File: public/japanese/manifest.json
Comment (HarveyPeachey) [inline public/japanese/manifest.json:35]: `public/japanese/images/icons/icon-180x180.png` is actually 246x246

**Review by chris-hinds — APPROVED** [review]
LGTM

Unfortunately you will need to comment out this this line in the Jenkinsfile https://github.com/bbc/simorgh/blob/latest/Jenkinsfile#L81

It is there a temporary measure to reduce the time it takes to upload static assets as this process was often timing out.

File: public/japanese/manifest.json
Comment (jimjohnsonrollings) [reply public/japanese/manifest.json:35]: Good spot

**Comment by jimjohnsonrollings** [comment]
Blocked pending updated Japanese PNG

File: public/japanese/manifest.json
Comment (jimjohnsonrollings) [reply public/japanese/manifest.json:35]: Should be sorted now, thanks

File: Jenkinsfile
Comment (HarveyPeachey) [inline Jenkinsfile:81]: Do we need to revert this once the images have been uploaded?

**Review by HarveyPeachey — APPROVED** [review]
LGTM just a small after thought

File: Jenkinsfile
Comment (jimjohnsonrollings) [reply Jenkinsfile:81]: Yeah - I'll write an issue for that.  Thanks

Comment (jimjohnsonrollings) [reply Jenkinsfile:81]: #7360

**Review by chris-hinds — APPROVED** [review]

**Pull request merged**
